src/vue/components/word-list/word-language-panel.vue

```diff
@@ -227,6 +236,20 @@ export default {
           return 0 // default state is unsorted
         }
       })
+    },
+    downloadList () {
+      const exportFields = [ 'targetWord', 'languageCode', 'important', 'currentSession', 'lemmasList' ]
```

Comment:
can we add the context target urls too?

Reply:
I have added context urls to a separate cell.